lib/tool_belt/commands/branch_docs_command.rb

```diff
@@ -0,0 +1,18 @@
+module ToolBelt
+  module Command
+    class BranchDocsCommand < Clamp::Command
+
+      parameter 'config_file', 'Release configuration file'
+
+      def execute
+        config = ToolBelt::Config.new(config_file)
+
```

Comment:
Maybe have this exit if the project is not katello here to inform other project users?